src/Factory/Consignment/ConsignmentFactory.php

```diff
@@ -113,7 +113,11 @@ public function fromOrder(
             $orderIso       = $countryService->getShippingCountryIso2($order);
             $carrierName    = CarrierService::getMyParcelCarrier($order->getIdCarrier())->getName();
 
-            if (CarrierPostNL::NAME === $carrierName && AbstractConsignment::CC_NL === $orderIso) {
+            if (
+                CarrierPostNL::NAME === $carrierName
```

Comment:
Dit is weer business logic herhalen, kijk eens hoe ver je komt met `$consignment->canHaveExtraOption()`?